tt_metal/common/core_descriptor.hpp

```diff
@@ -35,19 +35,16 @@ inline std::string get_core_descriptor_file(const tt::ARCH &arch) {
         wh_arch = "tt_metal/core_descriptors/";
         wh_arch += getenv("WH_ARCH_YAML");
     }
-    bool targeting_versim = false;
-#ifndef TT_METAL_VERSIM_DISABLED
-    targeting_versim = true;
-#endif
-
-    if (targeting_versim) {
+    
+    bool targeting_vcs = std::getenv("TT_METAL_SIMULATOR_EN") != nullptr; 
+    if (targeting_vcs) {
```

Comment:
this isn't VCS specific right?

Reply:
No, it shouldn't be I'll change to `TT_METAL_REMOTE_EN`

Reply:
changed